dlp/snippets/Stored_InfoType/stored_infotype_test.py

```diff
@@ -17,18 +17,23 @@
 from typing import Iterator
 import uuid
 
+import create_stored_infotype as create_si
+
 import google.api_core.exceptions
 import google.cloud.dlp_v2
 import google.cloud.exceptions
 import google.cloud.storage
+
+import inspect_with_stored_infotype as inspect_si
+
 import pytest
 
-import stored_infotype
+import update_stored_infotype as update_si
```

Comment:
nit: can you put imports from lines 20 and 27 (the tested samples) next to this line so they're all in a single block of imports?

Reply:
Doing so would cause linting errors, and hence they are intentionally put that way.